### PR TITLE
document remaining door assets

### PR DESCRIPTION
**Pull request by MegaIDK**
feel hella free to comment on the names cus yea

### CODE REVIEW

File: src/overlays/actors/ovl_En_Door/z_en_door.c [hunk 1]
```diff
@@ -369,17 +369,18 @@ static u8 sAnimCloseFrames[10] = {
...
     { gZoraHallDoorDL, gZoraHallDoorDL },
     { gSwampDoorDL, gSwampDoorDL },
     { gMagicHagPotionShopDoorDL, gMagicHagPotionShopDoorDL },
-    { object_wdor01_DL_000548, object_wdor01_DL_000548 }, // Lottery Shop / Curiosity Shop / Mayor's House Door
-    { object_wdor02_DL_000548, object_wdor02_DL_000548 }, // Trading Post / Post Office Door
-    { object_wdor03_DL_000548, object_wdor03_DL_000548 }, // Stockpot Inn & Swordsman's School Door
+    { gLotteryCuriosityShopMayorHouseDoorDL,
+      gLotteryCuriosityShopMayorHouseDoorDL }, // Lottery Shop / Curiosity Shop / Mayor's House Door
+    { gPostOfficeTradingPostDoorDL, gPostOfficeTradingPostDoorDL }, // Trading Post / Post Office Door
+    { gInnSchoolDoorDL, gInnSchoolDoorDL },                         // Stockpot Inn & Swordsman's School Door
```
Comment (AngheloAlf) [inline src/overlays/actors/ovl_En_Door/z_en_door.c:379]: Those comments are kinda redundant now, those could probably be removed now

File: assets/xml/objects/object_kinsta2_obj.xml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ﻿<Root>
+    <!--Object for Oceanside Spider House Door-->
     <File Name="object_kinsta2_obj" Segment="6">
-        <DList Name="object_kinsta2_obj_DL_000310" Offset="0x310" />
-        <Texture Name="object_kinsta2_obj_Tex_0004E8" OutName="tex_0004E8" Format="rgba16" Width="32" Height="64" Offset="0x4E8" />
+        <DList Name="OceansideSpiderHouseDoorDL" Offset="0x310" />
+        <Texture Name="OceansideSpiderHouseDoorTex" OutName="oceanside_spider_house_door" Format="rgba16" Width="32" Height="64" Offset="0x4E8" />
```
Comment (AngheloAlf) [inline assets/xml/objects/object_kinsta2_obj.xml:5]: Remember, the naming convention says assets' name should start with a `g`
```suggestion
        <DList Name="gOceansideSpiderHouseDoorDL" Offset="0x310" />
        <Texture Name="gOceansideSpiderHouseDoorTex" OutName="oceanside_spider_house_door" Format="rgba16" Width="32" Height="64" Offset="0x4E8" />
```

**Review by isghj5 — APPROVED** [review]

**Review by AngheloAlf — APPROVED** [review]

File: assets/xml/objects/object_dor01.xml [hunk 3]
```diff
@@ -1,7 +1,8 @@
 ﻿<Root>
+    <!-- Object for Astral Observatory / Marine Research Lab Door (Bland metal door)-->
```
Comment (engineer124) [inline assets/xml/objects/object_dor01.xml:2]:
```suggestion
    <!-- Object for Astral Observatory / Marine Research Lab Door (Bland metal door) -->
```

File: assets/xml/objects/object_kinsta2_obj.xml [hunk 4]
```diff
@@ -1,6 +1,7 @@
 ﻿<Root>
+    <!--Object for Oceanside Spider House Door-->
```
Comment (engineer124) [inline assets/xml/objects/object_kinsta2_obj.xml:2]:
```suggestion
    <!-- Object for Oceanside Spider House Door -->
```

**Review by engineer124 — APPROVED** [review]

**Pull request merged**